packages/core/admin/admin/src/pages/MarketplacePage/tests/index.test.js

```diff
@@ -18,8 +20,8 @@ import useNavigatorOnLine from '../../../hooks/useNavigatorOnLine';
 import MarketPlacePage from '../index';
 import server from './server';
 
-const GITHUB_STARS = 12;
-const WEEKLY_DOWNLOADS = 135;
+const GITHUB_STARS = 49130;
+const WEEKLY_DOWNLOADS = 7492;
```

Comment:
Instead of hardcoding these, it looks to me like a good place to use a regex [like this](https://regex101.com/r/WohnHo/1), that you can then use in your tests instead of the exact string